ci-operator/step-registry/cucushift/upgrade/run/cucushift-upgrade-run-commands.sh

```diff
@@ -1,129 +0,0 @@
-#!/bin/bash
```

Comment:
Instead of removing it here, we need a separate PR to remove this step. Because removing it here will make current upgrade job and working-in-progress jobs unusable.

Reply:
Updated. Recovered cucushift-upgrade-run step.